iron-dropdown.html

```diff
@@ -271,8 +272,7 @@
         },
 
         _cancelAnimations: function() {
-          this.cancelAnimation('open');
-          this.cancelAnimation('close');
+          this.cancelAnimation();
```

Comment:
Yeah, that's kind of strange, but perhaps a simplification of cancel's implementation?